src/js/index.js

```diff
@@ -0,0 +1,43 @@
+import { Calculator } from "./calculator.js";
+import { alertOnError } from "./utils/alert.js";
+import { isButtonElement } from "./utils/validators.js";
+
+const app = document.body.querySelector("#app");
+const calculator = new Calculator(app);
+
+const digitEventHandler = (e) => {
+  const target = e.target;
+  if (isButtonElement(target) && target.classList.contains("digit")) {
+    const digit = target.innerText;
+    if (calculator.state.operation === undefined) {
+      calculator.setFirstTerm(digit);
+    } else {
+      calculator.setSecondTerm(digit);
+    }
+  } else {
+    return;
+  }
```

Comment:
```suggestion
  const target = e.target;
  if (!isButtonElement(target) || !target.classList.contains("digit")) return;
  const digit = target.innerText;
  if (!calculator.state.operation) {
    calculator.setFirstTerm(digit);
    return;
  }
 calculator.setSecondTerm(digit);
```
else 구문은 불필요한 경우가 많답니다!
이런식으로 표현해주면 어떨까요?

Reply:
우선 조건이 많은 경우에는 부정 조건들을 활용한 것 보다 긍정 조건을 사용하는 것이 이해하는데 더 쉽다고 생각해서 위와 같이 작성했습니다. 그런데 [아래 코멘트](https://github.com/next-step/js-calculator/pull/127#discussion_r912482330)에 언급해주신 것처럼 검증 로직을 한 번 더 추상화하면 이해하기 쉬운 조건을 만들 수 있을 듯하여 변경해보겠습니다!

그리고 calculator.state.operation의 유무를 판단하여 term에 값을 설정하는 로직은 유무에 따라서 명확하게 해야하는 일이 두가지로 정해져야 한다고 생각하기 때문에 else문을 통해서 명시적으로 보여줘야 한다고 생각해서 위와 같이 작성했습니다! 

물론, 제안해주신 코드로도 로직을 이해하는 데 큰 문제가 없다고 생각하지만 제가 작성한 코드와 제안해주신 코드가 개개인의 코드 작성 견해인지 아니면 else문을 적게 사용하는 게 더 클린한(명시적인) 코드라는 근거 혹은 논리가 있는지 궁금합니다.:)

+) 혹시 제 문체가 딱딱하거나 공격적이라고 느껴지면 이야기해주세요. 부드럽게 작성하려고 하는데 다른 분들과 리뷰가 처음이라 이런 부분도 피드백 받고 개선하고 싶습니다 ㅎㅎ

Reply:
@ippowe 
- 마틴파울러가 제안한 객체지향 생활체조 원칙(https://jamie95.tistory.com/99)에 **else 쓰지 않기** 라는 내용이 존재한답니다!
- else를 사용하지 않을 경우 block을 최대한 얕게 유지할 수 있다는 장점이 있습니다 ㅎㅎ